server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/VariableWidthHistogramAggregationBuilder.java

```diff
@@ -149,12 +158,32 @@ protected ValuesSourceAggregatorFactory innerBuild(QueryShardContext queryShardC
                                                        ValuesSourceConfig config,
                                                        AggregatorFactory parent,
                                                        AggregatorFactories.Builder subFactoriesBuilder) throws IOException {
-
         Settings settings = queryShardContext.getIndexSettings().getNodeSettings();
         int maxBuckets = MultiBucketConsumerService.MAX_BUCKET_SETTING.get(settings);
         if (numBuckets > maxBuckets) {
```

Comment:
Side note for us to think about in the future: I wonder if we should escalate this sort of check to BucketsAggregator or something?  E.g. it doesn't make sense for any of the aggs to request > max_buckets and we can determine that up front at parsing.  Potentially save some heartache.

Less of an issue since we've increased the threshold but still.